bin/ctags.rb

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
```

Comment:
Thanks for pointing that out, I didn't really notice it.  It does no harm, but I can either remove it and modify the .gitignore, or I can leave it and modify code so it doesn't get into a gem.